src/Glpi/Asset/Capacity/HasSoftwaresCapacity.php

```diff
@@ -55,6 +56,12 @@ public function getIcon(): string
         return Software::getIcon();
     }
 
+    #[Override]
+    public function getDescription(): string
+    {
+        return __("List installed softwares found by automatic inventory");
```

Comment:
Not only from inventory
```suggestion
        return __("List installed softwares");
```

Reply:
Also, `software` is uncountable
```suggestion
        return __("List installed software");
```